spring-cloud-gcp-samples/spring-cloud-gcp-vision-ocr-sample/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
I think we're done with `http:`. I has to be `https:` across the board now. 😄

Reply:
Hmm, this one didn't quite work for me; I got a "URI not registered error" when trying https. I noticed that all the maven urls in the repo are still `http://`.